src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -891,6 +891,114 @@ To conform, the ``state`` entry must be an *Object* and can have the following o
+   * file_pattern: (string)
+   * number_of_trees: (number)
+   * tree_pattern: (string, default = "/")
+   * protocol/name: (string)
+   * protocol/version: (string)
+
+**Example of a Basic Root File:**
+
+  .. code:: yaml
+
+    blueprint_index: 
+      mesh: # Most entries under here were generated using generate_index()
+        state: 
+          cycle: 100
+          time: 3.1415
+          path: "state"
+          number_of_domains: 4
+        coordsets: 
+          coords: 
+            type: "uniform"
+            coord_system: 
+              axes: 
+                x: 
+                y: 
+                z: 
+              type: "cartesian"
+            path: "coordsets/coords"
+        topologies: 
+          mesh: 
+            type: "uniform"
+            coordset: "coords"
+            path: "topologies/mesh"
+        fields: 
+          density_000: 
+            number_of_components: 1
+            topology: "mesh"
+            association: "vertex"
+            path: "fields/density_000"
+            display_name: "density/mat0" # NOTE: This renames the field in VisIt
+          density_001: 
+            number_of_components: 1
+            topology: "mesh"
+            association: "vertex"
+            path: "fields/density_001"
+            display_name: "density/mat1"
+    # These entries specify the number and names of the files that make up the dataset.
+    number_of_files: 4
+    file_pattern: "bp/bp_%05d.hdf5"
+    number_of_trees: 4
+    tree_pattern: "/"
+    protocol: 
+      name: "hdf5"
+      version: "0.4.0"
+
+
+**Example code:**
+
+  .. code:: cpp
+
+    // Call on each of 4 MPI ranks.
+    conduit::Node mesh, bp_index;
+    conduit::blueprint::mesh::examples::braid("uniform", 10, 10, 10, mesh);
+    char domainFile[1024];
+    sprintf(domainFile, "./bp/bp_%04d.hdf5", rank);
+    conduit::relay::io::save(mesh, domainFile, "hdf5");
+
+    conduit::blueprint::mpi::mesh::generate_index(mesh,
+                                                  "",
+                                                  bp_index["blueprint_index/mesh"],
+                                                  MPI_COMM_WORLD);
+    bp_index["file_pattern"] = "./bp/bp_%04d.hdf5";
+    bp_index["number_of_files"] = 4;
+    bp_index["number_of_trees"] = 4;
+    bp_index["protocol/name"] = "hdf5";
+    bp_index["protocol/version"] = "0.4.0";
+    bp_index["tree_pattern"] = "/";
+    if(rank == 0)
+        conduit::relay::io::save(bp_index, "bp.root", "hdf5");
+
+
+Root files are needed to read Blueprint data into tools such as VisIt. At present, VisIt imposes a few caveats when reading Blueprint data:
+
+ * A root file is always required.
+ * The root file must be saved to hdf5, json, or yaml protocols.
+ * Individual Blueprint files that contain actual data must use the hdf5 protocol.
+ * If fields supply a ``display_name`` string then that name will be used instead of the field name. (e.g. display_name: "menu1/menu2/fieldname")
```

Comment:
```suggestion
 * If fields supply a ``display_name`` string then that name will be used instead of the field name (e.g. display_name: "menu1/menu2/fieldname").
```